ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -35,39 +35,278 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
       }
     ]
   };
-  function reportError(msg) {
-    if (msg) {
-      console.error(msg);
-    }
+  /**
+   * Common assertions for Add terms, use in term and taxonomy
+   * components tests
+   * @param {Object} assert - For assertions
```

Comment:
`Object` here should be set to `QUnit.assert` described in [their docs](https://api.qunitjs.com/QUnit.assert/)